packages/mechaml/mechaml.1.2.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer: "Yann Hamdaoui <yann.hamdaoui@gmail.com>"
+authors: "Yann Hamdaoui <yann.hamdaoui@gmail.com>"
+homepage: "https://github.com/yannham/mechaml"
+synopsis: "A functional web scraping library"
+description: """
+Mechaml is a functional web scraping library that allows to :
+ - Fetch web content
+ - Analyze, fill and submit HTML forms
+ - Handle cookies, headers and redirections
+"""
+bug-reports: "https://github.com/yannham/mechaml/issues"
+license: "LGPL v3"
+dev-repo: "git+https://github.com/yannham/mechaml.git"
+doc: "https://yannham.github.io/mechaml/"
+tags: ["web" "scraping" "mechanize" "http" "client"]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name ] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" {build & >= "1.8.0"}
```

Comment:
```suggestion
  "dune" {>= "1.8.0"}
```
Dune should not be tagged as a build dependency (see #14266)

Reply:
Thanks, it should be fixed now.